tests/etl/test_perf_data_adapters.py

```diff
@@ -155,6 +155,15 @@ def test_load_generic_data(self):
                                          subtest['value'],
                                          datetime.datetime.fromtimestamp(
                                              self.PUSH_TIMESTAMP))
+        summary_signature = PerformanceSignature.objects.get(
+            suite=perf_datum['suites'][0]['name'], test='')
+        subtest_signature = PerformanceSignature.objects.exclude(
+            suite=perf_datum['suites'][0]['name'], test='')
+        self.assertEqual(3, len(subtest_signature))
+        # Not sure why I can't just compare two Queryset list, and it always return error
+        # by self.assertEqual(subtest_signature, summary_signature.subtests.all())
+        for i in range(0, 3):
```

Comment:
Though I think this check could be better done using something like:

```
subtest_hashes = set([subtest_signature.signature_hash for signature in summary_signature.subtests.all()])
asset len(subtest_hashes) == 3
```

Note that I recently changed this unit test rather dramatically in bug 1228154, so you'll probably need to rewrite this code before landing.